spec/integrations/rake_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'webrick'
 require 'spec_helper'
 require 'json'
+require 'rake'
```

Comment:
This require is no longer needed, unless you want to add an additional expectation between lines 68-69 (something like `expect(result["events"][0]["device"]["runtimeVersions"]["rake"]).to eq(::Rake::VERSION)`)

Reply:
We removed that in place of a regex, so I'll remove the require.